rest/resource-server/src/main/resources/application.yml

```diff
@@ -34,5 +38,6 @@ security:
 
 sw360:
   thrift-server-url: ${SW360_THRIFT_SERVER_URL:http://localhost:8080}
+  couchdb-url: ${SW360_COUCHDB_URL:http://localhost:5984}
```

Comment:
Where the variable is used?

Reply:
Sorry, that is only necessary for the upload